pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1510,18 +1525,62 @@ func getNetworkProviderConfig(ctx context.Context, configManager ConfigManager,
+	// if the config map is found and the _network block contains an entry, that is returned for use in the network creation
+	if vpcCreateConfig.CidrBlock != nil && *vpcCreateConfig.CidrBlock != "" {
+		_, vpcCidr, err := net.ParseCIDR(*vpcCreateConfig.CidrBlock)
+		if err != nil {
+			return nil, errorUtil.Wrap(err, "failed to parse cidr block from _network strategy")
+		}
+		logger.Infof("found vpc cidr block %s in network strategy tier %s", vpcCidr.String(), tier)
+		return vpcCidr, nil
 	}
 
-	_, vpcCidr, err := net.ParseCIDR(*vpcCreateConfig.CidrBlock)
+	//if vpcCreateConfig.CidrBlock is nil or an empty string we can go ahead and set a sensible non overlapping default with a mask size of /26
+	defaultCIDR, err := n.getNonOverlappingDefaultCIDR(ctx)
 	if err != nil {
-		return nil, errorUtil.Wrap(err, "failed to parse cidr block from _network strategy")
+		return nil, errorUtil.Wrap(err, "failed to generate default CIDR")
 	}
-	cidrMask, _ := vpcCidr.Mask.Size()
 
-	logger.Infof("found vpc cidr block %s/%d in network strategy tier %s", vpcCidr.IP.String(), cidrMask, tier)
-	return vpcCidr, nil
+	// a default cidr is generated and updated to the config map, that is returned for use in the network creation
+	return defaultCIDR, nil
+}
+
+// getNonOverlappingDefaultCIDR returns a non overlapping cidr block based on the OpenShift vpc cidr block
+// the default mask is /26
+// for other masks the user is required to provide their own via config
+func (n *NetworkProvider) getNonOverlappingDefaultCIDR(ctx context.Context) (*net.IPNet, error) {
+	clusterVpc, err := getClusterVpc(ctx, n.Client, n.Ec2Api, n.Logger)
+	if err != nil {
+		errMsg := "failed to get cluster vpc for cidr block"
+		return nil, errorUtil.Wrap(err, errMsg)
+	}
+	//parse the cidr to an ipnet cidr in order to manipulate the ip
+	clusterCidrIp, clusterCidr, err := net.ParseCIDR(aws.StringValue(clusterVpc.CidrBlock))
+	if err != nil {
+		return nil, errorUtil.Wrap(err, "error parsing cluster cidr block")
+	}
+	ip := clusterCidrIp.To4()
+	for {
+		// increment the second byte of the ip by 4 bytes or by 1, e.g. if the ip is 10.0.0.0 then we want 10.1.0.0
+		// as the maximum cro allowed mask size is /16, we are using /26 for the default but the following default
+		// will apply in the case of /16 to /26
+		// do this repeatedly until a validCIDRRange is generated - in most cases the first one will satisfy
+		// if we've reached the maximum B range of 255, then increment the A range, e.g. 10.255.0.0/26 would result in 11.0.0.0/26
+		if ip[1] < 255 {
+			ip[1] = ip[1] + byte(1)
+		} else {
+			ip[1] = byte(0)
+			ip[0] = ip[0] + byte(1)
+		}
+
+		_, defaultCIDR, err := net.ParseCIDR(fmt.Sprintf("%s/%s", ip, defaultCIDRMask))
+		if err != nil {
+			return nil, errorUtil.Wrap(err, "error parsing standalone default cidr block")
+		}
+		if err = validateStandaloneCidrBlock(defaultCIDR, clusterCidr); err == nil {
```

Comment:
This was what I meant to use originally. 🤦‍♀️

Reply:
I don't think this will work, as the first none valid CIDR will return. I was thinking something like this : 

```
func (n *NetworkProvider) getNonOverlappingDefaultCIDR(ctx context.Context) (*net.IPNet, error) {
	// get cluster vpc
	clusterVpc, err := getClusterVpc(ctx, n.Client, n.Ec2Api, n.Logger)
	if err != nil {
		errMsg := "failed to get cluster vpc for cidr block"
		return nil, errorUtil.Wrap(err, errMsg)
	}

	// parse CIDR subnet
	clusterCidrIp, clusterCidr, err := net.ParseCIDR(aws.StringValue(clusterVpc.CidrBlock))
	if err != nil {
		return nil, errorUtil.Wrap(err, "error parsing cluster cidr block")
	}

	// find non overlapping default
	for i := 0; clusterCidr.Contains(incrementIP(clusterCidrIp, i)); i++ {

		// increment potential default
		nextNetwork := incrementIP(clusterCidrIp, i)

		// parse potential default to type CIDR
		_, defaultCIDR, err := net.ParseCIDR(nextNetwork.String())
		if err != nil {
			n.Logger.Errorf("error parsing default cidr '%s' skipping setting default : %v", nextNetwork.String(), err)
			continue
		}

		// validate the potential cidr
		if clusterCidr.Contains(defaultCIDR.IP) || defaultCIDR.Contains(clusterCidr.IP) {
			continue
		}

		return defaultCIDR, nil
	}
	// if loop ends return error
	return nil, errors.New("could not find default CIDR")
}
```
note, I didnt run this so it might not even compile :see_no_evil: , just we should check for every possible potential default, if none is found we should error